boards/fanke/fk750m1_vbt6/fk750m1_vbt6_defconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) zack jiang <1125934312@qq.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable UART
+CONFIG_SERIAL=y
+
+# Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable Pinctrl
+CONFIG_PINCTRL=y
+
```

Comment:
```suggestion
```

Reply:
Why do we turn off PINCTRL by default here?